.ci/docker/build.sh

```diff
@@ -316,7 +324,6 @@ fi
 # Build image
 docker build \
        --no-cache \
-       --progress=plain \
```

Comment:
Why are you changing this?

Reply:
Latest docker seems discarding the option, at least on docker 24.0.5. Actually, I saw the error message:
```
DEPRECATED: The legacy builder is deprecated and will be removed in a future release.
            Install the buildx component to build images with BuildKit:
            https://docs.docker.com/go/buildx/

unknown flag: --progress
```

Reply:
Reverted